public/locales/en-US/send.ftl

```diff
@@ -117,13 +117,10 @@ passwordIsSet = Password set
 maxPasswordLength = Maximum password length: { $length }
 # A short status message shown when there was an error setting the password
 passwordSetError = This password could not be set
-#
-#
-#
-# shared ->
-#
-#
-#
+
+# Denotes strings shared across versions
+ [[ shared ]]
```

Comment:
This is not a supported syntax anymore. We don't have section names, we can have section comments (with two hashes).

```
 ## Strings shared across different versions of Send
```